### ISSUE TITLE
Remove spawn-sync dep

### ISSUE THREAD

**Issue by wesleytodd** [timoxley/npm-run#5]
Is it possible to remove that dep in favor of using the [built in one](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options)?

I understand backwards compat issues, BUT let me tell you my story:

I am using the linter happiness (a fork of standard).  I added it to the `package.json`'s of my company modules, about 100 of them.  We are also using linklocal for development and a recursive link via bulk.  Turns out that the npm install in all these directories takes like an hour.  Here is the dependency tree that is leading here:

```
node_modules/happiness/node_modules/happiness-format/node_modules/esformatter/node_modules/npm-run/node_modules/spawn-sync
```

`spawn-sync` is a c extension, so gets compiled for every `npm install` that is run.  I didnt spend the time to figure out how many times it is run, but I started it at like 9am and now it is 12 and it is still running :)

I had some time, so figured I would open an issue here.....

**Comment by timoxley** [comment]
I think this is a good call. Please submit a PR! :dancer:

**Comment by wesleytodd** [comment]
Will do!!  Probably on Monday :)

**Comment by wesleytodd** [comment]
Closed in #6
